packages/shared-lib/src/core/model/Timeline.ts

```diff
@@ -2,6 +2,19 @@ import { unprotectString, protectString } from '../../lib/protectedString'
 import { TSR } from '../../tsr'
 import { MappingsHash, PeripheralDeviceId, StudioId, TimelineBlob, TimelineHash } from './Ids'
 
+export interface TimelineObjectAbSessionInfo {
```

Comment:
I think it would be nice if this had a bit of an explanation on what this AB logic means by a "Session".